ladder/teams/README.md

```diff
@@ -0,0 +1,51 @@
+# How Cilium Org Teams Are Managed
+Each Cilium org team that wishes to self-manage its team membership can do so using this directory.
+
+Each team can establish a file in this directory managed by the CodeOwners system with the assistance of 
+the Org admins (or duly deputized members of the org tasked with helping initiating team self-management). 
+
+On merge into the main branch (after appropriate approvals and review) GitHub automation 
+will propogate updates from the team files into the Cilium Organization's team membership using the GitHub API.
+
+## Are You a Team Member or a Mentor? Choose, but Choose Wisely.
+With membership comes both priviledges and responsibilities.  One of the responsibilities of being on an active Cilium team member is 
+being available for automatic pull request review assignment for files owned by that team. 
+
+Over time as contributors transition from one team to another inside the project, while not active on a team any longer, 
+their historic experience may be needed to help review proposed changes with functionality they have insight into.  In these cases these 
+individuals can be added to the team's mentor list. Mentors are no longer part of the auto-assigned PR review rotation, but they can still keep team 
+approval access.
```

Comment:
I'd probably add something also to this effect:
```suggestion
approval access. Mentors of a team are expected to provide guidance to others in the team and seek reviews they should participate in for the code owner area.
```

Reply:
To be fair, I think this is largely covered already by the [reviewer ladder description](https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md#reviewer), but it becomes more critical for mentors to keep an eye out to help if GitHub is not auto-assigning reviews to them.